### PR TITLE
add tutorial to convert sdf to usd

### PR DESCRIPTION
**Pull request by koonpeng**
# 🎉 New feature

## Summary

Short example of using `sdf2usd`.

~TODO: Seems like some poses are wrong, will update the screenshot after it is fixed.~ Fixed
omni-create

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: convert_sdf_to_usd/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+```bash
+sdf2usd /usr/share/ignition/ignition-gazebo6/worlds/shapes.sdf shapes.usda
+```
+
+*Note: The format of the output depends on the extension, use `.usd` to create a USD binary file instead.*
+
+Test the result by opening it in Omniverse Create.
+
+<!--TODO: Bug in sdf2usd? Some shapes are clipping through the plane-->
```
Comment (ahcorde) [inline convert_sdf_to_usd/tutorial.md:42]: Fixed here https://github.com/ignitionrobotics/sdformat/pull/870, isn't it ?

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: convert_sdf_to_usd/tutorial.md (hunk 1 above)
Comment (koonpeng) [reply convert_sdf_to_usd/tutorial.md:42]: updated the ss

**Review by ahcorde — CHANGES_REQUESTED** [review]
Yo need to add this file in the `manifest.xml`

**Comment by koonpeng** [comment]
> Yo need to add this file in the `manifest.xml`

updated manifest

**Review by ahcorde — APPROVED** [review]
@azeey do you mind to review this ?

File: convert_sdf_to_usd/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Convert SDF to USD
+
+## Prerequisites
+
+* sdformat with USD support (see the [sdformat installation instructions](../install/tutorial_12.md#USD))
```
Comment (azeey) [inline convert_sdf_to_usd/tutorial.md:5]: The link is not working in the preview. Can you try:
```suggestion
* sdformat with USD support (see the [sdformat installation instructions](/tutorials?tut=install&ver=>%3D+12))
```

**Review by azeey — COMMENTED** [review]
Looks good! A question about where the tutorial belongs and a couple of tweaks on links and images. Here's the preview: http://sdformat.org/tutorials?tut=convert_sdf_to_usd&cat=specification&branch=usd-convert

File: manifest.xml [hunk 3]
```diff
@@ -239,6 +253,7 @@
       </description>
       <tutorials>
         <tutorial>convention_tutorial</tutorial>
+        <tutorial>convert_sdf_to_usd</tutorial>
```
[the remark this answers is not in the document]
Comment (ahcorde) [reply manifest.xml:256]: friendly ping @scpeters

[the remark this answers is not in the document]
Comment (scpeters) [reply manifest.xml:256]: sorry for the delay. How about the "For Developers" section down below? we could also make a new section about converting between formats

[the remark this answers is not in the document]
Comment (koonpeng) [reply manifest.xml]: I updated the category (I think? Not sure how the manifest works).

[the remark this answers is not in the document]
Comment (scpeters) [reply manifest.xml]: yep, you did it right. You can preview here:

* http://sdformat.org/tutorials?cat=developers&branch=usd-convert

* http://sdformat.org/tutorials?tut=convert_sdf_to_usd&cat=developers&branch=usd-convert

File: manifest.xml [hunk 4]
```diff
@@ -127,6 +127,20 @@
       <skill>intermediate</skill>
     </tutorial>
 
+    <tutorial title='Convert SDF to USD' ref='convert_sdf_to_usd' type='documentation'>
+      <markdown version='1.7'>convert_sdf_to_usd/tutorial.md</markdown>
```
Comment (scpeters) [inline manifest.xml:131]: I would change the version to `12` to correspond to libsdformat12

File: convert_sdf_to_usd/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Convert SDF to USD
+
+## Prerequisites
+
+* sdformat with USD support (see the [sdformat installation instructions](/tutorials?tut=install&ver=>%3D+12))
```
Comment (scpeters) [inline convert_sdf_to_usd/tutorial.md:5]: can you remove `&ver=>%3D+12` from this link? the link appears broken for me like that. Tested from:

* http://sdformat.org/tutorials?tut=convert_sdf_to_usd&cat=developers&branch=usd-convert

**Review by scpeters — APPROVED** [review]

**Pull request merged**